app/models/upvs/message_template.rb

```diff
@@ -62,6 +62,9 @@ def create_message(author:, box:, recipient_name:, recipient_uri:)
       create_form_object(message)
     end
 
+    EventBus.publish(:message_thread_created, message.thread) if message.thread.previously_new_record?
+    EventBus.publish(:message_created, message)
```

Comment:
uplne sa mi nepozdava ze tu mame takto vela miest na vytvaranie draftov. Je to naozaj tak?

Reply:
Ano, lebo:
UPVS draft vies vytvorit:
- cez API
- z balika cez GUI
- z templatu cez GUI

FS draft:
- z XML cez GUI.

Reply:
Pripomen mi preco tie eventy nemame v nejakom after_create_commit hooku. Mari sa mi ze sme to tak mali ale pre nieco sme to dali prec.

Reply:
Nevedeli sme to napasovat na callback, lebo to v skutocnosti nie je viazane iba na vytvorenie samotnej spravy, ale spravy ako celku (t.j. sprava + vsetky objekty s ich obsahom - kedze v tie pravidla velakrat riesia prave obsah objektov).